.github/workflows/osx.yml

```diff
@@ -18,8 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
-        - 13
-        - 14
```

Comment:
I think we want to exclude OSX13 - AppleClang and OSX14 - AppleClang, but keep OSX13 - clang and OSX14 - clang

Reply:
Fixed 
![image](https://github.com/user-attachments/assets/95514d2f-a980-4ea8-95f5-2c87e82ae2d5)